downstream/titles/hub/getting-started/master.adoc

```diff
@@ -20,6 +20,8 @@ include::automation-hub/proc-create-api-token.adoc[leveloffset=+1]
 
 The API token is now available to use to configure {HubName} as your default collections server or when uploading collections using the `ansible-galaxy` command line tool.
```

Comment:
This sentence is a bit awkward, suggest rewording to:

The API token is now available for configuring {HubName} as your default collections server or uploading collections using the `ansible-galaxy` command line tool.

Reply:
Thanks for the peer review @dcdacosta. I've made the changes. Can you pls approve and merge?